get_all_manifests.sh

```diff
@@ -6,19 +6,19 @@ GITHUB_URL="https://github.com/"
 # component: notebook, dsp, kserve, dashbaord, cf/ray/kueue, trustyai, modelmesh, modelregistry.
 # in the format of "repo-org:repo-name:branch-name:source-folder:target-folder".
 declare -A COMPONENT_MANIFESTS=(
-    ["codeflare"]="opendatahub-io:codeflare-operator:main:config:codeflare"
-    ["ray"]="opendatahub-io:kuberay:master:ray-operator/config:ray"
-    ["kueue"]="opendatahub-io:kueue:dev:config:kueue"
-    ["data-science-pipelines-operator"]="opendatahub-io:data-science-pipelines-operator:main:config:data-science-pipelines-operator"
-    ["odh-dashboard"]="opendatahub-io:odh-dashboard:incubation:manifests:dashboard"
-    ["kf-notebook-controller"]="opendatahub-io:kubeflow:v1.7-branch:components/notebook-controller/config:odh-notebook-controller/kf-notebook-controller"
-    ["odh-notebook-controller"]="opendatahub-io:kubeflow:v1.7-branch:components/odh-notebook-controller/config:odh-notebook-controller/odh-notebook-controller"
-    ["notebooks"]="opendatahub-io:notebooks:main:manifests:notebooks"
-    ["trustyai"]="trustyai-explainability:trustyai-service-operator:release/1.10.2:config:trustyai-service-operator"
-    ["model-mesh"]="opendatahub-io:modelmesh-serving:release-0.11.1:config:model-mesh"
-    ["odh-model-controller"]="opendatahub-io:odh-model-controller:release-0.11.1:config:odh-model-controller"
-    ["kserve"]="opendatahub-io:kserve:release-v0.11.1:config:kserve"
-    ["modelregistry"]="opendatahub-io:model-registry-operator:main:config:model-registry-operator"
+    ["codeflare"]="opendatahub-io:codeflare-operator:v1.1.0-rhoai:config:codeflare"
```

Comment:
i do not think CFO will use this branch for their release

Reply:
@astefanutti Can you confirm the release version for codeflare

Reply:
@VaishnaviHire latest version for CodeFlare is https://github.com/opendatahub-io/codeflare-operator/releases/tag/v1.2.0.

Reply:
https://github.com/opendatahub-io/opendatahub-community/issues/146#issuecomment-1989086281